### DISCUSSION TITLE
Community chats?

### DISCUSSION THREAD

**Discussion by Manueljlin** [uxbox/uxbox#529] [General]
Something like a Telegram or Matrix room, or even a Discord group would be great imo :D

**Comment by hirunatan** [comment]
We have a brand new chatroom in gitter:

https://gitter.im/penpot/community

Feel free to enter and comment anything.

**Reply by KaKi87** [reply]
No more chat :

![Screenshot from 2022-09-22 23-06-10](https://user-images.githubusercontent.com/21284089/191851031-62ca2792-f079-40f8-ae95-dd0ead1c08d3.png)

So, the question is relevant again.

**Comment by yorqat** [comment]
Hi, would anyone please start a discord server? Penpot is too much of a hotpot to not have an accessible forum

**Reply by dm17** [reply]
Who has the authority to do so? Quick chats would definitely be useful.

**Reply by VNRARA** [reply]
Year later, still nada?
